src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -584,12 +584,34 @@ private bool SetFrame(float left, float top, float right, float bottom, bool ind
                         // the size of the owner view must be reset even the ExcludeLayouting is true.
                         if (Owner.HeightSpecification < 0 || Owner.WidthSpecification < 0)
                         {
-                            Owner.SetSize(right - left, bottom - top);
+                            if (Owner.WidthResizePolicy == ResizePolicyType.DimensionDependency && Owner.HeightResizePolicy != ResizePolicyType.DimensionDependency)
+                            {
```

Comment:
I tested now how this works.
If we set width or height to 0, it don't change resize policy of the dimension.
DALi depends to set size zero.

But, what I am wondering is.... when we set Layout View's SizeHeight again, then Owner.SizeWidth is not zero and changes resize policy to Fixed.
And, that time, the size of each entries not changed following newly set SizeHeight.
Is it right to determine entries' size only once?

Reply:
Thank you for your comment.
You are right..
If SizeHeight is set only once, it works fine.
However, if SizeHeight is changed, then the SizeWidth becomes Fixed..

I will find another way to fix this.